pom.xml

```diff
@@ -462,7 +462,7 @@
             <dependency>
                 <groupId>com.clickhouse</groupId>
                 <artifactId>clickhouse-jdbc</artifactId>
-                <version>0.6.3</version>
+                <version>0.7.1-patch1</version>
```

Comment:
Can we wait for their official release or is `patch1` is their official release ?

Reply:
This change might allow us to rename the commit message and release-notes as we are updating the client as well

Reply:
Looks like `patch` suffix is an official way in their release process.